descriptors/ModuleDescriptor-template.json

```diff
@@ -15,8 +15,28 @@
             "consortia.tenants.collection.get"
           ],
           "modulePermissions": []
-        }
 
+        },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/consortia/user-tenants",
+          "permissionsRequired": [
+            "consortia.user-tenants.collection.get"
+          ],
+          "modulePermissions": []
+        },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/consortia/user-tenants/{associationId}",
+          "permissionsRequired": [
+            "consortia.user-tenant.get"
```

Comment:
I guess this should be `consortia.user-tenants.item.get`.

Reply:
Thanks, I will fix